member.go

```diff
@@ -67,8 +68,15 @@ func (c *Client) Member(nick string) (member *Member, err error) {
 	return
 }
 
-func (m *Member) Boards() (boards []Board, err error) {
-	body, err := m.client.Get("/member/" + m.Id + "/boards")
+func (m *Member) Boards(field ...string) (boards []Board, err error) {
+	fields := ""
+	if len(field) == 0 {
+		fields = "name,shortUrl"
```

Comment:
So, this basically breaks all existing apps relying on all Board fields, right? How about having all fields by default and let your app pass 
```go
boards, err := member.Boards("name", "shortUrl")
```
instead?